homeassistant/components/litterrobot/sensor.py

```diff
@@ -62,7 +62,7 @@ class LitterRobotSleepTimeSensor(LitterRobotPropertySensor):
     def native_value(self) -> str | None:
```

Comment:
Does this typing need adjustment here and in the parent class?

Reply:
Yeah. I guess it's not checked. I'll update.